### ISSUE TITLE
6. Ontbrekend HTML lang-attribuut  (API's template A11y)

### ISSUE THREAD

**Issue by TessaViergever** [frameless/developer.overheid.nl#108]
👪 Related issue: [#115](https://github.com/frameless/developer.overheid.nl/issues/115)

### Betrokken pagina's

[Zoek API's](https://rijkshuisstijl-community-templates-3losjges9-nl-design-system.vercel.app/developer-overheid/api-overzicht)

### Beschrijf het probleem

De html lang-attribuut ontbreekt nog steeds.

### Relevante code

```html
`<html class="utrecht-root">`
```

### WCAG-richtlijn

WCAG 3.1.1 Language of Page Level A

### Voeg een afbeelding toe (indien relevant)

![Screenshot of the developer tools which show the page lacks a lang attribute](https://github.com/user-attachments/assets/87c118ac-8746-4caf-a7db-71b72f64b94e)

### Referentienummer

TBD
